multinode-demo/bootstrap-validator.sh

```diff
@@ -61,6 +61,15 @@ while [[ -n $1 ]]; do
     elif [[ $1 = --enable-rpc-bigtable-ledger-storage ]]; then
       args+=("$1")
       shift
+    elif [[ $1 = --tpu-use-quic ]]; then
```

Comment:
Correct @CriesofCarrots. This is a heuristic controls to avoid extreme configurations of the two parameters. It is not a full rate limiting implementation. That will require another PR to address.